ToolBox/packages.config

```diff
@@ -5,7 +5,7 @@
   <package id="Common.Logging.Core" version="3.4.1" targetFramework="net462" />
   <package id="DotNetZip" version="1.13.3" targetFramework="net452" />
   <package id="IKVM" version="8.1.5717.0" targetFramework="net452" />
-  <package id="IQFeed.CSharpApiClient" version="2.5.1" targetFramework="net462" />
+  <package id="IQFeed.CSharpApiClient" version="2.6.0" targetFramework="net462" />
```

Comment:
Believe the `csproj` file also has to be updated, suggest updating using the nuget package manager of VS